warp10/src/main/java/io/warp10/json/JsonUtils.java

```diff
@@ -31,11 +31,6 @@
 import com.fasterxml.jackson.databind.ser.BeanSerializer;
 import com.fasterxml.jackson.databind.ser.BeanSerializerModifier;
 import com.fasterxml.jackson.databind.ser.impl.UnknownSerializer;
```

Comment:
This part of the PR doesn't have much to do with the rest!

Reply:
I know, it will be a part of a second PR.